strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/environment/EnvironmentInfoController.java

```diff
@@ -85,14 +85,13 @@ public ResponseEntity getEnvironmentInfo()
 
         strongboxInfo.add(new EnvironmentInfo("storages", String.valueOf(storagesCount)));
 
-        Long usersCount = userService.getUsers()
+        Long usersCount = userService.getUsers(null, null)
```

Comment:
@anki2189 such an implementation is not correct. On the other hand, the real user count will run slowly, so I suggest to drop `usersCount` from `strongboxInfo`. @steve-todorov will you mind?

Reply:
I agree the implementation is wrong - it was meant to count the `users` from the `yaml` configuration file and will not work in the current context. Right now, I suggest we always return `0` and add a `TODO: Implement a user counter` comment above. In the long run we are probably going to rely on this information for licensing purposes.

Reply:
Fixed.